infra/build/functions/sync/main_test.py

```diff
@@ -18,15 +18,16 @@
 
 from collections import namedtuple
 import os
-import unittest
 import subprocess
 import threading
+import unittest
+import requests
```

Comment:
fix the order here to be alphabetical.

Reply:
When i put it in in alphabetical order earlier it gave me linting errors:
`infra/build/functions/sync/main_test.py:22:0: C0411: standard import "import subprocess" should be placed before "import requests" (wrong-import-order)
infra/build/functions/sync/main_test.py:23:0: C0411: standard import "import threading" should be placed before "import requests" (wrong-import-order)
infra/build/functions/sync/main_test.py:24:0: C0411: standard import "import unittest" should be placed before "import requests" (wrong-import-order)
`